ksql-engine/src/test/java/io/confluent/ksql/KsqlContextTestUtil.java

```diff
@@ -77,7 +78,11 @@ public static KsqlContext create(
         serviceContext,
         ksqlConfig,
         engine,
-        (ec, sc) -> InjectorChain.of(schemaInjector, new DefaultTopicInjector(ec))
+        (ec, sc) -> InjectorChain.of(
```

Comment:
I kept it this way because I know that @spena was refactoring it now so I wanted fewer jitters around it. I don't mind doing it now if you feel strongly about it.

Reply:
Makes sense - but we should clean it up at some point. It just makes it more confusing for others and harder to test / mock etc in the long run.

Reply:
totally agree